example/README.md

```diff
@@ -29,18 +29,18 @@ NOTE: For this to work you need your default credentials to be credentials for t
 
 ### Additional deployment steps
 
-An S3 Access lambda is needed in the us-west-1 region to run the tests. To initially create the lambda, run:
+An S3 Access lambda is needed in the us-west-2 region to run the integration tests. To initially create the lambda, run:
 
 ```
-aws lambda create-function --region us-west-1  --function-name <STACK>-S3AccessTest --zip-file fileb://app/build/cloudformation/<ZIP>-S3AccessTest.zip  --role arn:aws:iam::<AWS_ACCOUNT_ID>:role/<PREFIX>-lambda-processing  --handler index.handler --runtime nodejs6.10 --profile ngap-sandbox
+aws lambda create-function --region us-west-2  --function-name <STACK>-S3AccessTest --zip-file fileb://app/build/cloudformation/<ZIP>-S3AccessTest.zip  --role arn:aws:iam::<AWS_ACCOUNT_ID>:role/<PREFIX>-lambda-processing  --handler index.handler --runtime nodejs8.10 --profile ngap-sandbox
```

Comment:
ngap explicitly denies access in us-west-1 so we need to use us-west-2

Reply:
👍